afy/camera_selector.py

```diff
@@ -98,14 +100,17 @@ def select_camera(cam_frames, window="Camera selector"):
 
 
 if __name__ == '__main__':
-    with open('config.yaml', 'r') as f:
-        config = yaml.load(f, Loader=yaml.FullLoader)
+    parent_dir = os.path.basename(os.path.dirname(os.getcwd()))
+    file_location = ['Desktop', 'Downloads', 'Documents', parent_dir, '', 'Programs']
+    for loc in file_location:
+        yml_file_path = os.path.join(os.path.expanduser('~'), loc, 'config.yaml')
+        if os.path.exists(yml_file_path):
+            with open(yml_file_path, "r") as f:
+                config = yaml.load(f, Loader=yaml.FullLoader)
 
```

Comment:
I think it might be a good idea to check that a config was loaded, and print a helpful error message if it wasn't found in any of the locations. Since the new code doesn't fail when it can't find the config file, it would probably lead to a more cryptic error later.